### DISCUSSION TITLE
WixBundleInstalled is always 0 (false)

### DISCUSSION THREAD

**Discussion by SamOxigen** [wixtoolset/issues#6910] [Questions]
This built-in variable returns false always from the engine, and I have few major problems with it.
1. I cannot re-install/repair. My all attempts will create new bundle install entries, and ironically, will never be detected on my next attempt.
While wix setup projects can have fixed GUID number for uninstall registry key, this feature is not available for the bootstrapper bundle, and upon prior installation detection failure, engine create next bundle uninstall registry entry every time I run the bootstrapper.
I noticed also that this, WixBundleInstalled cannot be overwritten.

Any clue why burner fails to detect the bundle install state?

**Comment by SamOxigen** [comment]
I'm giving more details about a bundle hoping someone can see where I'm missing something that eventually causing this error, namely; after an initial/clean installation Wix fails to detect the previous installation (WixBundleInstalled is always 0, false), and with every subsequential installation run, the installer creates a new uninstall registry entry.
I do have "my way" of detecting a previous installation, and, currently, I cancel/exist the bootstrapper engine upon detecting the previous installation.
While this is ok for no upgrade/downgrade scenarios, as I said, I'm locked with "clean" install scenario.
Or, I have to uninstall some packages during my subsequential install then install a newer/older version of it. However, this solution does not address duplicate  uninstall registry entries.

`	<Bundle Name="My Company Bundle"
			Version="1.0.0.0"
			Manufacturer="My Manufacturer"
			IconSourceFile="App.ico"
			UpgradeCode="59D74F13-2C2A-4092-9DF4-A2052961B229"
			HelpTelephone="800-700-6000"
			HelpUrl="http://support.mycompany.com"
			Copyright="©2022 My Company, LLC."
			DisableModify="yes"`
			
So, why I'm getting duplicate uninstall registry entries when running the installer on machines where I have run the installer already?

**Reply by robmen** [reply]
A bundle log file will inform much. Also, I don't know if this is relevant, but every time you build a bundle it will create a brand new bundle.

**Reply by SamOxigen** [reply]
I'll re-run the same installer over again and post the log with some altered private/security info around this afternoon. Thanks for the reply.

**Reply by robmen** [reply]
I'd recommend trying to interpret the log file yourself first. With some study, there is a good chance you'll see the issue yourself.
